UnitedIncome/components
# [APP-4006] Storybook MDX Docs

### Description
> Please provide a brief description of your changes below.

Migrates _all_ component library docs to `mdx` format. I've cleaned up a lot of the documentation, and added a prop table to each one. There's likely a few spelling errors, but once this merges we can fix them as we see them. The generator has also been updated to support this new format, along with the linter/formatter.

I've disabled `addon-info` and `addon-readme` as part of this as they are no longer required. I've also ran the `Organize Import` tool that VSCode has on all `.js` files in the repo as a general cleanup item. At some point in the future I'll figure out a way to lint that, but for now we can manually do it.

Stories and docs are still defined separately for now, and code samples are provided beneath the story. Eventually we may change how this works based on how DocPages evolves but this is still a step in the right direction for now. 

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

* Load storybook.
* Click on the `Docs` tab.
* Click through each story and observe the new docs.

### Comments
> Any additional comments that will aid with the review process.

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-4006)
